WcaOnRails/lib/custom_i18n_scanner.rb

```diff
@@ -0,0 +1,69 @@
+require 'i18n/tasks/scanners/file_scanner'
+class WCAFileScanner < I18n::Tasks::Scanners::FileScanner
+  include I18n::Tasks::Scanners::RelativeKeys
+  include I18n::Tasks::Scanners::OccurrenceFromPosition
+
+  def active_record?(model)
+    # FIXME: Find a way to determine this dynamically ?
+    model != "contact"
+  end
+
+  def extract_model_attr(key)
+    model_key = key.gsub(/^(\w+)\..*$/, '\1').singularize
+    # Some dirty tricks to assign the detected field to the "correct" model
+    # For most controllers the forms modify a model which has the same name
+    # as the controller, however for some of them they modified a single different
+    # model (eg: the `admin` forms modify `person`s, and only that model).
+    model_name = case model_key
+                 when "devise" then "user"
+                 when "admin" then "person"
+                 when "oauth" then "doorkeeper/application"
+                 else model_key
+                 end
+    attribute = key.gsub(/.*\.(\w+)$/, '\1')
+    [model_name, attribute]
+  end
+
+  # @return [Array<[absolute key, Results::Occurrence]>]
+  def scan_file(path)
+    text = read_file(path)
+    retval = []
+    # First scan for inputs (or labels for inputs) that are not hidden
+    text.scan(/^\s*<%= f.(input|label|input_field) :(\w+)(?!.*hidden)(.*)%>/) do |match|
```

Comment:
Yes it [does](https://ruby-doc.org/core-2.2.0/String.html#method-i-scan).
Originally I indeed followed the `i18n-tasks` example :)